vm/src/stdlib/time_module.rs

```diff
@@ -58,6 +58,18 @@ pub fn get_time() -> f64 {
     Date::now() / 1000.0
 }
 
+#[cfg(any(not(target_arch = "wasm32"), target_os = "wasi"))]
```

Comment:
i didn't know wasm has confusing discussion about it.
https://stackoverflow.com/questions/6002808/is-there-any-way-to-get-current-time-in-nanoseconds-using-javascript